tests/integration/api_v3/http/transactions.test.js

```diff
@@ -528,20 +546,20 @@ describe('Transactions API', () => {
 			expect(response.meta).toMap(metaSchema);
 		});
 
-		it('returns 400 BAD REQUEST with unsupported params', async () => {
+		it('should throw error when called with unsupported params', async () => {
 			const expectedStatusCode = 400;
 			const response = await api.get(`${endpoint}?address=${refTransaction.sender.address}&nonce=${Number(refTransaction.nonce) - 1}`, expectedStatusCode);
 			expect(response).toMap(badRequestSchema);
 		});
 
-		it('returns 200 OK when queried with transactionID and non-zero offset', async () => {
+		it('should throw error when queried with transactionID and non-zero offset', async () => {
```

Comment:
Just to maintain consistency.
```suggestion
		it('should throw error when called with transactionID and non-zero offset', async () => {
```